cmake/projects/SFML/hunter.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2016-2017, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    SFML
+    VERSION
+    2.4.2
+    URL
+    "https://github.com/SFML/SFML/archive/2.4.2.tar.gz"
+    SHA1
+	275C68B51B8509BA90EDA6D5DEF791B497F1F777
+)
+
+hunter_cmake_args(
+    SFML
+    CMAKE_ARGS
+        SFML_BUILD_AUDIO=NO
+)
+
```

Comment:
So... Here I must add
`hunter_add_package(OpenAL)`

And fork SFML repo for patching it's CMakeLists? (to add the following line to it: `find_package(OpenAL CONFIG REQUIRED)`)?

Reply:
You add `hunter_add_package(OpenAL)` in the CMakeLists.txt file.  Every `find_package(FOO)` should be paired with `hunter_add_package(FOO CONFIG REQUIRED)` (options vary).  When hunter is enabled, then it will build and install `FOO` and the `find_package(FOO)` command will work.  If you disable hunter, `hunter_add_package()` calls becomes hoops, and cmake will look for the package somewhere on your system.

Reply:
> And fork SFML repo for patching it's CMakeLists? (to add the following line to it: find_package(OpenAL CONFIG REQUIRED))?

You can create an issue asking for a fork of the project in `hunter-packages` where a minimal set of CMake changes can be applied.

Reply:
It's already forked: https://github.com/hunter-packages/SFML

I've created branch `hunter-2.4.2` from `2.4.2` tag, please send patches there.